chart/values.yaml

```diff
@@ -178,6 +178,17 @@ apis:
         cpu: "50m"
         # -- Memory requests for rest
         memory: "32Mi"
+    # Rest service parameters defines how the rest service is exposed
+    service:
+      # K8s service type
+      type: ClusterIP
+
+      # Ports from where rest endpoints are accessible from outside cluster, only valid if type is NodePort
+      nodePorts:
```

Comment:
Should these lines be commented out by default?

Reply:
Yes IMO.

Reply:
No, these values are also configurable. It will not work if we comment them.

Reply:
Seeing similar configuration for Loki and etcd as well. Even though, by default, service type is clusterIp, there is a nodePort value as well.

Reply:
No, we have an if conditional in the template that would add/remove this based on the types.
This in similar to what other chart providers follow.:- ref etcd https://github.com/bitnami/charts/blob/610f860fd8197f6f1dc1906511b56a96df42dac9/bitnami/etcd/values.yaml#L517

Reply:
Having said that, we can still comment out these lines by default, specifying in the comment that "Uncomment below lines if serviceType is nodePort" - @Abhinandan-Purkait

Reply:
Okay. Shall I comment them for `etcd` and `loki-stack` as well, so that its similar throughout.

Reply:
I don't think we should comment them out. They're there as the default in case node port is used.

Reply:
As discussed, looks good, no need to comment.